build_artifacts/v2/v2.0/v2.0.0/v2.0.0-beta/dirs/etc/maxdome/git_clone.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+METADATA=/opt/ml/metadata/resource-metadata.json
+
+# Extract the required fields from meta data stored in opt/ml/metadata.
+dataZoneProjectRepositoryName=$(grep -o '"DataZoneProjectRepositoryName":"[^"]*"' "$METADATA" | cut -d ':' -f 2 | tr -d '"')
+
+#User directory on which the user will have full access to . 
+MD_USER="sagemaker-user"
+USER_DIRECTORY="/home/${MD_USER}"
+
+# Configuring git credential helper to use AWS creds when connecting to Codecommit Repo. 
+git config --global credential.helper '!aws codecommit credential-helper $@' 
```

Comment:
We included https://github.com/aws/git-remote-codecommit into SMD, so we can directly use  git clone codecommit::$AWS_REGION://repo /local/dir